### DISCUSSION TITLE
Using QLayout with router views

### DISCUSSION THREAD

**Discussion by bigfatwhale** [elimintz/justpy#310] [Q&A] [answered]
Hi, 

I'm trying to implement something like, 

<img width="935" alt="image" src="https://user-images.githubusercontent.com/1858007/137120963-46545143-2ed9-4fd1-a2f1-24e1870308f4.png">

using QLayouts : https://quasar.dev/layout/routing-with-layouts-and-pages

How do i use the router-view in Quasar so that for example, clicking on the Portfolio item in the drawer will show some new content in the center of the page?

Thanks!

**Comment by elimintz** [comment, marked as answer]
With JustPy you can't use the router-views but you don't need to. You simply change the content of the page based on the input of the user, but you do it in python on the backend (there is no frontend in JustPy). 

Here is an example where the content on the page is changed based on user interaction:
https://justpy.io/tutorial/handling_events/#example-3-event-changes-elements-on-page

Here is a complete example with Quasar's layout:
https://github.com/elimintz/justpy/blob/master/examples/dogs.py

**Reply by bigfatwhale** [reply]
Thanks for the prompt reply! I'm slowing getting the hang of how to do things in Justpy :)
